docs/api/advanced/snapshot.mdx

```diff
@@ -7,22 +7,89 @@ description: 'Create a snapshot of current state'
 
 # snapshot
 
-#### A snapshot takes a proxy and returns a immutable object, unwrapped from the proxy.
+snapshot takes a proxy and returns an immutable object, unwrapped from the proxy.
 
-Immutability is achieved by deeply freezing the object. In sequential snapshot calls, when the values in the proxy have not changed, a pointer to the same object is returned. This allows for shallow comparison in render functions, preventing spurious renders. Snapshots also throw promises, making them work with React Suspense.
+Immutability is achieved by deeply copying & freezing the object. In sequential snapshot calls, when the values in the proxy have not changed, a pointer to the same object is returned. This allows for shallow comparison in render functions, preventing spurious renders. Snapshots also throw promises, making them work with React Suspense.
 
 ```js
 import { proxy, snapshot } from 'valtio'
 
 const store = proxy({ name: 'Mika' })
-const unwrappedStore = snapshot(store) // unwrapped store is the original object, unproxied
+const unwrappedStore = snapshot(store) // unwrapped store is a copy of the original object, unproxied
 const anotherUnwrappedStore = snapshot(store)
 console.log(unwrappedStore === anotherUnwrappedStore) // true
 store.name = 'Hanna'
 const yetAnotherUnwrappedStore = snapshot(store)
 console.log(unwrappedStore === yetAnotherUnwrappedStore) // false
 ```
 
+#### Copy on Write
```

Comment:
I realize that line 12 is already saying the same thing, but dunno, personally it took me awhile to understand / get it, so feel this is useful. Happy to drop/edit.